frontend/src/features/permits/pages/ShoppingCart/ShoppingCartPage.tsx

```diff
@@ -349,6 +350,13 @@ export const ShoppingCartPage = () => {
   return (
     <div className="shopping-cart-page">
       <Box className="shopping-cart-page__left-container">
+        <div className="shopping-cart-page__info">
+          <p className="info__body">
+            Have questions? Please contact the Provincial Permit Centre.
+            Toll-free: <strong>1-800-559-9688</strong> or Email:{" "}
```

Comment:
Please use the TOLL_FREE_NUMBER constant inside frontend/src/common/constants/constants.ts instead